src/Razor/test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/BasicComponent_DesignTime.codegen.cs

```diff
@@ -15,7 +15,7 @@ public class AspNetCore_d3c3d6059615673cb46fc4974164d61eabadb890 : Microsoft.Asp
         ((System.Action)(() => {
 #nullable restore
 #line 1 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/BasicComponent.cshtml"
-IDisposable __typeHelper = default(IDisposable);
+IDisposable __typeHelper = default(IDisposable)!;
```

Comment:
So this helps the editor fo-sho. What happens when you do a build? (runtime code gen)

Reply:
It's hidden because at runtime the other code for this method is not in a pragma. Basically, the runtime code is never nullability enforced for directives anyhow because it has no line pragmas.

Reply:
That alone is probably its own issue though because it means any errors coming from directives today aren't properly mapped.

Reply:
Yeah that issue in particular is coming up a lot in Blazor. Users aren't getting line mappings for mistakes in `@inject` which then takes to you to the generated file from VS.

It looks like this answers my question tho.